python/docs/source/improve_docs.rst

```diff
@@ -0,0 +1,7 @@
+Something Missing?
+------------------
+
+If something is missing in the documentation or if you found some part confusing, please file
+an `issue <https://github.com/VowpalWabbit/vowpal_wabbit/issues/new/choose>`_ with your suggestions for
+improvement. Use the "Documentation Improvement" template.
```

Comment:
This issue template does not exist

Reply:
Okay, I will remove it or I was thinking of making it as we have for the bugs report.

Reply:
I think an issue template for docs improvement is a good idea

Reply:
I have made the template https://github.com/VowpalWabbit/vowpal_wabbit/pull/2361
please have a look and tell if any modification is required so that it can be merged and I can point that template here.